docs/current_docs/quickstart/index.mdx

```diff
@@ -1,28 +1,40 @@
 ---
 slug: /quickstart
-hide_table_of_contents: true
+hide_table_of_contents: false
 title: "Introduction"
 ---
 
 # Quickstart
 
 Welcome to Dagger, a programmable tool that lets you replace your software project's artisanal scripts with a modern API and cross-language scripting engine.
 
-This tutorial walks you, step by step, through the basics of using the Dagger CLI, developing your first Dagger Function, and packaging it into a module.
+Dagger can help you create a consistent, reliable development and testing environment on your local host and ensure that everything your CI can do, your local environment can do too.
 
-## What you will learn
+## Topics
 
-In this tutorial, you will learn how to:
+This quickstart introduces you to:
 
-- Install the Dagger CLI
-- Call a Dagger Function
-- Understand Dagger Functions
-- Use Dagger Functions with directories and containers
-- Chain Dagger Functions together into a pipeline
-- Write your own Dagger Function
+- The Dagger API, which lets you use core primitives to stitch together powerful pipelines out of containers, artifacts, network services, and secrets – all in type-safe Go, Python or TypeScript.
+- Dagger Functions, which let you encapsulate code into software components which can be loaded into any Dagger Engine, and called by other Dagger Functions.
+- The Dagger CLI, which lets you create, inspect and invoke Dagger Functions from the command line.
+- The Daggerverse, which lets you search and consume publicly available Dagger Functions.
+
+## Goal
+
+After going through this quickstart, you will have the tools and skills to create a CI pipeline to test, build and release your code, and start replacing your custom CI scripts with reusable Dagger Functions from the Daggerverse or that you create yourself.
+
+## Approach
+
+This quickstart is structured as a hands-on workshop, in which you will create a CI pipeline for an example application, then run and test it locally. It discusses two approaches:
+
+1. Using the core Dagger API: To begin, you will build a CI pipeline for the example application using only the Dagger API. This is an important first step to learn how Dagger works and to understand key concepts and techniques. It is also critical knowledge for when no Daggerverse module exists for your requirements and you need to start from scratch.
+
+1. Using a Daggerverse module: At a later stage, you will refactor the same CI pipeline using a module from the Daggerverse. This alternative approach introduces you to the Daggerverse, a key component of the Dagger ecosystem. It also demonstrates the speed and efficiency benefits of reusing existing Dagger modules when constructing your CI pipeline.
 
 ## Requirements
 
-To use this tutorial, you only need the Dagger CLI installed on your machine.
+This quickstart will take you approximately 30 minutes to complete.
```

Comment:
The target was 10 min. However there is a lot to cover, so I couldn't get it down to under this time limit. For the moment, I'm being explicit about the time required. I'm open to ideas to condense/cut some material.